manifest.json

```diff
@@ -181,6 +181,18 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "dockerfile": "src/debian/stretch/source-build",
+              "os": "linux",
+              "osVersion": "stretch",
+              "tags": {
+                "debian-stretch-source-build-$(System:DockerfileGitCommitSha)-$(System:TimeStamp)": {}
```

Comment:
For this tag and the other 18.04 tag, can you be sure to swap the position of the commit SHA and timestamp?  See #275